ledger-tool/src/main.rs

```diff
@@ -910,6 +910,12 @@ fn main() {
         .validator(is_parsable::<usize>)
         .takes_value(true)
         .help("How many accounts to add to stress the system. Accounts are ignored in operations related to correctness.");
+    let accounts_filler_size = Arg::with_name("accounts_filler_size")
+        .long("accounts-filler-size")
+        .value_name("BYTES")
+        .validator(is_parsable::<usize>)
+        .takes_value(true)
+        .help("Size per filler account in bytes. [default: 0]");
```

Comment:
prefer `clap::Arg::default_value()` to maintaining the help string and implementation independently